rocketpy/motors/liquid_motor.py

```diff
@@ -463,21 +463,19 @@ def add_tank(self, tank, position):
         self.positioned_tanks.append({"tank": tank, "position": position})
         reset_funcified_methods(self)
 
-    def draw(self):
-        """Draw a representation of the LiquidMotor."""
-        self.plots.draw()
+    def draw(self, filename=None):
+        """Draw a representation of the LiquidMotor.
 
-    def info(self):
-        """Prints out basic data about the Motor."""
-        self.prints.all()
-        self.plots.thrust()
-
-    def all_info(self):
-        """Prints out all data and graphs available about the Motor.
```

Comment:
deleted info and all_info ?

Reply:
They are already present in the `Motor` parent class. I see no need for repeating them unless a child requires a different implementation. They were all the same.